internal/builders/gradle/create_attestation.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# "version" and "attestations" fields:
+echo -e -n "{\n  \"version\": 1,\n  \"attestations\": [" >> "$SLSA_OUTPUTS_ARTIFACTS_FILE"
+
+NUM_JAR_FILES=$(find ./release-files-for-slsa -type f | wc -l)
+COUNTER=1
```

Comment:
nit: same lowercase comment as above.
```suggestion
num_jar_files=$(find ./release-files-for-slsa -type f | wc -l)
counter=1
```